labelbox/pagination.py

```diff
@@ -134,7 +134,7 @@ class _CursorPagination(_Pagination):
     def __init__(self, cursor_path: List[str], *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.cursor_path = cursor_path
-        self.next_cursor: Optional[Any] = None
+        self.next_cursor: Optional[Any] = kwargs.get('params', {}).get('from')
```

Comment:
This change makes the code a bit obscure. To enhance readability and maintainability, what do you think about explicitly adding params to the arguments of the constructor?

Reply:
I did add it in the past... but the parent init mypy had issues with it... ok let me look at it again

Reply:
I had it like you suggested, but had issues similar to https://stackoverflow.com/questions/21764770/typeerror-got-multiple-values-for-argument with mypy.. so I would prob have to 're-add' it back into kwargs before calling the parent constructor. Let me see